SharpSource/SharpSource.CodeFixes/Diagnostics/AsyncMethodWithVoidReturnTypeCodeFix.cs

```diff
@@ -39,20 +40,20 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
             diagnostic);
     }
 
-    private static Task<Document> ChangeReturnTypeAsync(Document document, SyntaxNode methodDeclaration, SyntaxNode root)
+    private static async Task<Document> ChangeReturnTypeAsync(Document document, SyntaxNode methodDeclaration, SyntaxNode root)
     {
+        var model = await document.GetSemanticModelAsync();
+        var methodSymbol = model!.GetSymbolInfo((methodDeclaration as MethodDeclarationSyntax)!.ReturnType)!.Symbol!;
+        var annotation = SymbolAnnotation.Create(methodSymbol);
```

Comment:
The symbol here looks incorrect. You need a symbol annotation for `Task` (which is the new return type that you want to include imports to).

I'd probably avoid semantic model calls and just do `SymbolAnnotation.Create("System.Threading.Tasks.Task")`

Reply:
That worked a treat! One code fix introduced a newline-related issue but I've seen that in other places as well. I think I can fix it by moving to the VerifyCS approach so I'll investigate that one further. Many thanks for the suggestion here 🙏